dotcom-rendering/src/model/extract-ga.ts

```diff
@@ -58,6 +58,6 @@ export const extract = (data: CAPIArticleType): GADataType => ({
 	toneIds: filterTags(data.tags, 'Tone'),
 	seriesId: filterTags(data.tags, 'Series'),
 	isHosted: 'false', // TODO - This is missing from the Frontend Data model
-	edition: data.editionId.toLowerCase() as Edition,
+	edition: data.editionId.toLowerCase() as EditionId,
```

Comment:
This is odd, as `toLowerCase` would never produce a value that could be an `EditionId`. (I know this is pre-existing, just thought it work pointing out as it's breaking the type system).

Reply:
I've actually just removed the EditionId from this type because as far as I could tell it was both wrong, and unused in 'typed' land anyway.